cpp/ql/test/query-tests/Likely Bugs/Memory Management/StrncpyFlippedArgs/test.cpp

```diff
@@ -93,3 +95,17 @@ void test8(char x[], char y[]) {
   // that it will be a false positive if we report it.
   strncpy(x, y, 32);
 }
+
+void test9()
+{
+	wchar_t buf1[10];
+	wchar_t buf2[20];
+	const wchar_t *str = L"01234567890123456789";
+
+	wcsxfrm_l(buf1, str, sizeof(buf1), nullptr);					// BAD (but not a StrncpyFlippedArgs bug)
```

Comment:
> (but not a StrncpyFlippedArgs bug)

I'm not sure what you're saying here, the query appears to have flagged this line.

Reply:
Yeah, good point... `test9()` is a copy of `test2()` pretty much, including the comments, and has the same behaviour. Namely, that line is flagged because (I'm guessing) it falls under the ["Fixed sized case"](https://github.com/github/codeql/blob/a144828266e87d4a4cdac02f4b546245453ce11e/cpp/ql/src/Likely%20Bugs/Memory%20Management/StrncpyFlippedArgs.ql#L88), which is not exactly a case of "flipped" args. At least, that's how I interpreted the original comment.